### DISCUSSION TITLE
What version of ChatGPT is this using?

### DISCUSSION THREAD

**Discussion by 0xAskar** [llm-workflow-engine/llm-workflow-engine#43] [Q&A] [answered]
What version of ChatGPT is this using / what dataset is it trained on for a lack of better words?

For instance, I gave it a silly question of who is a better futbol player, Messi or Ronaldo, and when describing their accomplishments, seemed to be behind in the past few years events? Is this a chatGPT version trained on a 2019/2020 dataset?

Thanks.

**Comment by 0xAskar** [comment]
Hmm, I just realized the current chatgpt also gives the same answer which was cut off by 2021. So I'm assuming that doesn't include data in 2021?

**Comment by Mahmoud-ModelMe** [comment, marked as answer]
This is just a wrapper around the webbrowser version of chatGPT. It was trained up to data in 2021. It is also usually not very good with facts.
